codestyle-config/pom.xml

```diff
@@ -23,15 +23,17 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
-    <groupId>org.apache.apex</groupId>
-    <artifactId>apex</artifactId>
-    <version>3.3.0-incubating</version>
+    <groupId>org.apache</groupId>
+    <artifactId>apache</artifactId>
+    <version>18</version>
+    <relativePath/>
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
+  <groupId>org.apache.apex</groupId>
   <artifactId>apex-codestyle-config</artifactId>
-  <name>Apache Apex (incubating) Codestyle Config</name>
-  <version>1.0.0-incubating</version>
+  <name>Apache Apex Codestyle Config</name>
+  <version>1.1.0-SNAPSHOT</version>
```

Comment:
I see, so a source release of core could contain a snapshot version of the
codestyle-config module?

Thanks

On Fri, Sep 2, 2016 at 11:36 AM, Thomas Weise <notifications@github.com>
wrote:

> In codestyle-config/pom.xml
> <https://github.com/apache/apex-core/pull/380#discussion_r77393247>:
>
> >    <artifactId>apex-codestyle-config</artifactId>
> > -  <name>Apache Apex (incubating) Codestyle Config</name>
> > -  <version>1.0.0-incubating</version>
> > +  <name>Apache Apex Codestyle Config</name>
> > +  <version>1.1.0-SNAPSHOT</version>
>
> No, it is released separately.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/apex-core/pull/380/files/08954cdf30331f1e4faf6de9e2cdcf54c046fe1d#r77393247>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AENqu_vTXdiKKKM7mRf6tQf0zpESACfnks5qmGyagaJpZM4Jz-Ur>
> .
>